src/VisualStudio/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Roslyn.VisualStudio.Test.Utilities.Common;
+
+namespace Roslyn.VisualStudio.Test.Utilities.InProcess
+{
+    public class Editor_InProc : InProcComponent
```

Comment:
A bit confused on this this type. Many members are instance members, but it looks like they all call `ExecuteOnActiveView` which appears to be static? Should the rest of those become static as well?